packages/portal-proto/src/features/cohortComparison/CohortComparison.tsx

```diff
@@ -79,9 +79,13 @@ const CohortComparison: React.FC<CohortComparisonProps> = ({
   useEffect(() => {
     createPrimaryCaseSet({
       filters: buildCohortGqlOperator(cohorts.primary_cohort.filter) ?? {},
+      intent: "portal",
+      set_type: "ephemeral",
     });
     createComparisonCaseSet({
       filters: buildCohortGqlOperator(cohorts.comparison_cohort.filter) ?? {},
+      intent: "portal",
+      set_type: "ephemeral",
     });
```

Comment:
<img width="1435" alt="Screenshot 2024-04-05 at 2 47 11 PM" src="https://github.com/NCI-GDC/gdc-frontend-framework/assets/4624053/0f6b62e7-b7d2-448e-8dce-af0bf728d3ec">
venn diagram and survival plot analysis

Reply:
prob. needs @craigrbarnes to decide, but ephemeral does seem right (assuming "instant" would cause issues if the set were to get deleted somehow and the user does some survival plot /venn diagram activities). or is this the one where sets gets re-created whenever we detect they are not present?

Reply:
No, but it is the same as set operations where the created sets are used in multiple request.

Reply:
ephemeral